docs/supporting-artifacts/0020-LMS-Tables.sql

```diff
@@ -1,16 +1,16 @@
-/*
- * NOTE: the tables below have been modified from the output generated by
- * MetaEd: the primary key columns have been set to IDENTITY.
- */
```

Comment:
At this point I prefer putting the raw MetaEd output into this `supporting-artifact` directory, and then only modify them when copying over into the LMS DS Loader utility.

Also there was a recent update to MetaEd to support putting the file headers in the output when run with "Alliance Mode" enabled. This is very appropriate for these files.